resources/views/profile/edit.blade.php

```diff
@@ -175,6 +175,13 @@
                 },
                 profileUpdate() {
                     this.resetErrors();
+                    if (@json($enabled2FA) &&  this.global2FAEnabled.length === 0) {
+                      let message = 'The Two Step Authentication Method has not been set. ' +
+                      'Please contact your administrator.';
+                      // User has not enabled two-factor authentication correctly
+                      ProcessMaker.alert(this.$t($message, 'warning'));
```

Comment:
The 'warning' string is in a wrong place as a second parameter of $t() instead of alert()
```suggestion
                      ProcessMaker.alert(this.$t($message), 'warning');
```

Reply:
The change has been made